code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -453,6 +453,14 @@
 	else if(M.reagents.has_reagent(/datum/reagent/water, needs_metabolizing = FALSE))
 		M.reagents.remove_reagent(/datum/reagent/water, 0.5 * absorption_coeff)
 		. += power * 0.5
+	else if(ishuman(M))
+		var/mob/living/carbon/human/bloodyhooman = M
+		if(bloodyhooman.dna?.species?.exotic_blood == /datum/reagent/water/holywater && !(NOBLOOD in bloodyhooman.dna.species.species_traits) && bloodyhooman.blood_volume >= BLOOD_VOLUME_NORMAL + 0.5*absorption_coeff) //do NOT make this able to lower your water-blood level below BLOOD_VOLUME_NORMAL, or self-resp will be able to keep this active all the time on its own.
+			bloodyhooman.blood_volume -= 0.5*absorption_coeff
+			. += power * 0.75 //technically, if someone with holy water as their exotic blood has normal water in their bloodstream, the normal water will be prioritized over their holy water. that's an extreme edge case that would require way too much logic to cover properly, though.
+		else if(bloodyhooman.dna?.species?.exotic_blood == /datum/reagent/water && !(NOBLOOD in bloodyhooman.dna.species.species_traits) && bloodyhooman.blood_volume >= BLOOD_VOLUME_NORMAL + 0.5*absorption_coeff)
+			bloodyhooman.blood_volume -= 0.5*absorption_coeff
+			. += power * 0.5
```

Comment:
All a virologist needs to do is release the hydration virus and podpeople healing goes into overdrive?
Please set the healing to 0.25 for holy and 0.1 for water

Plasma can stay the same since it's rarely used and also requires the podperson to intentionally design themselves, instead of randomly getting insane healing when a viro released a hydration virus (which happens often enough)